libs/apps/python.go

```diff
@@ -0,0 +1,127 @@
+package apps
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const DEBUG_PORT = "5678"
+
+var defaultLibraries = []string{
```

Comment:
Can you comment where or how you pull an updated list?

This will grow stale.

Reply:
Added the link and comment